Marlin/src/lcd/menu/menu_tune.cpp

```diff
@@ -178,21 +178,21 @@ void menu_tune() {
   //
   #if FAN_COUNT > 0
     #if HAS_FAN0
-      MENU_MULTIPLIER_ITEM_EDIT_CALLBACK(uint8, MSG_FAN_SPEED FAN_SPEED_1_SUFFIX, &thermalManager.lcd_tmpfan_speed[0], 0, 255, thermalManager.lcd_setFanSpeed0);
+      MENU_MULTIPLIER_ITEM_EDIT_CALLBACK(percent, MSG_FAN_SPEED FAN_SPEED_1_SUFFIX, &thermalManager.lcd_tmpfan_speed[0], 0, 255, thermalManager.lcd_setFanSpeed0);
```

Comment:
I’m somewhat uncomfortable with this menu behavior. As far as I know this is the only edit type that conceals the real value being edited. Meanwhile, the G-code M106 takes the correct 0-255 values as documented. Sure, the user interface is for dummies and shows percent under the fan, but I kind of like the parity between M106 and the UI.

Reply:
okay then we leave it as it is.